rhc-ose-ansible/playbooks/openstack/terminate.yml

```diff
@@ -8,7 +8,7 @@
 - hosts: localhost
   gather_facts: false
   vars:
-    ansible_ssh_user: root
+    ansible_ssh_user: cloud-user
```

Comment:
This is only valid for environments where `cloud-user` is indeed the account to use - i.e.: OS1. Should probably just set this to the `rhc_ose_ssh_user` variable in the inventory file.